renovate.json

```diff
@@ -40,4 +49,4 @@
       "versioningTemplate": "semver-coerced"
     }
   ]
-}
+}
```

Comment:
I think it's JSON-specific, because it's only `renovate.json` that I have this problem in any repository.  I'm kinda fine with it because it's more machine-read than not.